### ISSUE TITLE
[findrepe] some pipes block the process

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#6]
```
some pipes like .me-tv/.me-tv/video.fifo block the process when trying to
read it

a viable solution is not-reading 0-size files.

```

Original issue reported on code.google.com by `flik...@gmail.com` on 22 Jun 2009 at 9:09

**Comment by GoogleCodeExporter** [comment]
```
fixed, version 0.6.0
```

Original comment by `flik...@gmail.com` on 4 Jul 2009 at 12:31
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 2 Jan 2010 at 10:23
- Changed title: **[findrepe] some pipes block the process**
